SimPEG/directives.py

```diff
@@ -106,7 +106,7 @@ def prob(self):
            Assuming that dmisfit is always a ComboObjectiveFunction,
            return a list of problems for each dmisfit [prob1, prob2, ...]
         """
-        return [objfcts.prob for objfcts in self.dmisfit.objfcts]
+        return [objfcts.simulation for objfcts in self.dmisfit.objfcts]
```

Comment:
thanks for all of these! Would you mind also updating line 104 here to be `def simulation(self)` and then for helping with backwards compatibility, we can add a 
```python

@property
def prob(self):
        warnings.warn(
            "The prob property will be depreciated in favor of simulation. "
            "Please update your code accordingly"
        )
        return self.simulation
```

Reply:
Can do! Will get on this shortly.

Reply:
@lheagy Done and Done!